test/commands/ps/type.js

```diff
@@ -6,6 +6,11 @@ const cmd = commands.find((c) => c.topic === 'ps' && c.command === 'type')
 const nock = require('nock')
 const expect = require('chai').expect
 
+let app = {
```

Comment:
Is is a little weird that this is being used for both the shield and non-shield cases below? maybe it should be a function so callers can do just plain `app()` or `app({shield: true})`?